release/ray_release/command_runner/command_runner.py

```diff
@@ -85,7 +87,12 @@ def run_prepare_command(
         Command runners may choose to run this differently than the
         test command.
         """
-        return self.run_command(command, env, timeout)
+        return exponential_backoff_retry(
```

Comment:
what is this change for?

Reply:
Somehow in the new stack it needs waiting for a while (1-3s) before it's ready to be used